terraform/modules/region/cloudwatch.tf

```diff
@@ -198,7 +198,7 @@ resource "aws_cloudwatch_metric_alarm" "uid_service_high_request_rate" {
   alarm_description         = "An abnormally high rate of requests detected in the ${local.environment_name} UID service."
   alarm_name                = "${local.environment_name}-uid-service-high-request-rate"
   comparison_operator       = "GreaterThanUpperThreshold"
-  datapoints_to_alarm       = 5
+  datapoints_to_alarm       = 65
   evaluation_periods        = 5
```

Comment:
I think this is saying that 65 out of the last 5 snapshots must be breaching, which means it will _never_ fail?

Reply:
The anomaly detection model seems to be struggling to catch up to the new baseline, and I don't think we can reset it. So maybe we could disable alarm actions for now until it's caught up?